specification.md

```diff
@@ -30,13 +30,13 @@ Keys that are applicable for all services.
 
 Keys for connecting to AWS services.
 
-| Key                   | Json Type | Default        | Example                  |
-|-----------------------|-----------|----------------|--------------------------|
-| `awsAccessKeyId`      | string    |                | `"ABCDABCDABCDABCD1234"` |
-| `awsSecretAccessKey`  | string    |                |                          |
-| `awsSessionToken`     | string    |                |                          |
-| `awsRegion`           | string    |                | `"eu-central-1"`         |
-| `awsEndpointOverride` | string    | _AWS endpoint_ | `"s3.my-company.de"`     |
+| Key                   | Json Type | Default        | Example                            |
+|-----------------------|-----------|----------------|------------------------------------|
+| `awsAccessKeyId`      | string    |                | `"ABCDABCDABCDABCD1234"`           |
+| `awsSecretAccessKey`  | string    |                |                                    |
+| `awsSessionToken`     | string    |                |                                    |
+| `awsRegion`           | string    |                | `"eu-central-1"`                   |
+| `awsEndpointOverride` | string    | _AWS endpoint_ | `"s3.my-company.de"` (no protocol) |
```

Comment:
`no protocol` may be not very clear at first read.

Maybe add asteriks, and short description at the end of page. `It should be without protocol, for example, http or https`.